packages/app-mobile/ios/Joplin.xcodeproj/project.pbxproj

```diff
@@ -700,14 +700,14 @@
 				COPY_PHASE_STRIP = NO;
 				CURRENT_PROJECT_VERSION = 83;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				DEVELOPMENT_TEAM = A9BXAFS6CT;
+				DEVELOPMENT_TEAM = 2FT8HR2Y7S;
 				GCC_C_LANGUAGE_STANDARD = gnu11;
 				INFOPLIST_FILE = ShareExtension/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @executable_path/../../Frameworks";
 				MARKETING_VERSION = 12.10.0;
 				MTL_FAST_MATH = YES;
-				PRODUCT_BUNDLE_IDENTIFIER = net.cozic.joplin.ShareExtension;
+				PRODUCT_BUNDLE_IDENTIFIER = net.personalizedrefrigerator.joplin.ShareExtension;
```

Comment:
That should probably not be changed

Reply:
Oops! Sorry! That was for my own custom build!!! I'll fix that!!!